opentrons_drivers/common/methods.py

```diff
@@ -0,0 +1,121 @@
+import time
+from typing import Callable
+from opentrons.protocol_api.labware import Well
+from opentrons.protocol_api.instrument_context import InstrumentContext
+import opentrons_drivers.common.helpers as help
+
+LIQUID_METHODS: dict[str, Callable[..., object]] = {}
+register_liquid_method = help.make_registry_decorator(LIQUID_METHODS)
+
+"""
+    All liquid transfer methods must have the same base signature:
+    pipette: InstrumentContext, to: Well, fr: Well, amount: float
+    
+    Anything else can be passed as keyword arguments
+    e.g. rate, iterations, etc. as **kwargs
+
+"""
+@register_liquid_method("basic_liquid_transfer")
+def basic_liquid_transfer(pipette: InstrumentContext, 
+                     to: Well, fr: Well, 
+                     amount: float) -> None:
+    """
+    Basic liquid transfer method.
+
+    Parameters:
+        pipette (InstrumentContext): Pipette to use.
+        to (Well): Target well.
+        fr (Well): Source well.
+        amount (float): Total volume to transfer.
+
+    Returns:
+        None
+    """
+    amts = help.liquid_batching(pipette, amount)
+    for a in amts:
+        pipette.aspirate(a, fr)
+        pipette.air_gap(20)
```

Comment:
20 seems like a magic number, could be a variable? Either global or for the function with a default.